eksconfig/cluster-loader.go

```diff
@@ -6,6 +6,7 @@ type ClusterLoaderSpec struct {
 	TestConfigUris []string `json:"testConfigUris,omitempty"`
 	TestParams     []string `json:"testParams,omitempty"`
 	S3Uri          string   `json:"s3Uri,omitempty"`
+	TestOverrides  []string `json:"testOverrides,omitempty"`
```

Comment:
I'd strongly recommend either a string (cl2's convention) or a map (best go data type to represent the data). What you're doing is inventing a new data type to represent a file and then hardcoding some newlines into it.

Reply:
I think having an array as input would make more sense to the users as opposed to having them format their overrides into one string, or even as a map. Having it as an array of strings is more akin to how one would create their own overrides file for CL2, which I would argue makes the most sense.

Reply:
Ok, SGTM.